src/main/tray/tray-menu-item/tray-menu-item-injection-token.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectionToken } from "@ogre-tools/injectable";
+import type { IComputedValue } from "mobx";
+import type { LensMainExtension } from "../../../extensions/lens-main-extension";
+
+export interface TrayMenuItem {
+  id: string;
+  parentId: string;
+  orderNumber: number;
+  enabled: IComputedValue<boolean>;
+  visible: IComputedValue<boolean>;
+
+  label?: string;
```

Comment:
Because e.g. separator does not have label. Even better would be separate implementations for different types of tray items. That would comply more with Open Closed Principle and Interface Segregation Principle. But for this, it would be overkill.